### DISCUSSION TITLE
GetComponentsInChildren 함수가 작동하지 않습니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#388] [5. Q&A]
GetComponentsInChildren 함수가 작동하지 않는 이유가 있나요?

Start() 함수에서도 해봤고, 
Update() 함수에서도 해봤지만 ... null 에러만 나옵니다.

**Comment by minjujuu** [comment]
간단한 예제이긴 하지만 다음과 같이 하면 정상 동작 확인되어 공유 드려봅니다

`import { GameObject, Transform } from 'UnityEngine'
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class Test extends ZepetoScriptBehaviour {

    public parent : GameObject;
    Start() {    
        for(let i=0; i<this.parent.transform.childCount; i++) {
            console.log(`${this.parent.GetComponentsInChildren<Transform>()[i+1].gameObject.name}`);
        }
    }

    Update() {
        for(let i=0; i<this.parent.transform.childCount; i++) {
            console.log(`${this.parent.GetComponentsInChildren<Transform>()[i+1].gameObject.name}`);
        }
    }
}`

위 코드 parent 오브젝트에 다음 Test 오브젝트를 연결해주었습니다
![image](https://user-images.githubusercontent.com/79794414/154029607-64893b43-fc8c-4cbe-b0fe-0b7a49dc9333.png)

![image](https://user-images.githubusercontent.com/79794414/154029650-d53e23bb-c026-49eb-824a-7626d81923a7.png)

**Reply by peterkangdev** [reply]
감사합니다.
